### PR TITLE
Set duration to n/a if startTime is n/a

### LINKED ISSUE #4472: fly shows the wrong duration for cancelled builds
Looks like its computing time since the epoch? Probably fly is using some kind of integer type to represent those timestamps, so if the start time is not set, it defaults to "0 milliseconds since the epoch".

Screen Shot 2019-09-20 at 11 07 07 AM

I think it would make more sense to see `n/a` in the duration field, or even `cancelled` in the duration field.

### PR DESCRIPTION
**Pull request by iamjarvo**
Fixes #4472

Screen Shot 2019-09-26 at 2 05 43 PM

### CODE REVIEW

File: fly/commands/builds.go [hunk 1]
```diff
@@ -254,7 +254,7 @@ func populateTimeCells(startTime time.Time, endTime time.Time) (ui.TableCell, ui
 		durationCell.Contents = endTime.Sub(startTime).String()
 	}
 
-	if startTime == zeroTime && endTime == zeroTime {
+	if startTime == zeroTime && endTime == zeroTime || startTime == zeroTime {
```
Comment (pivotal-jamie-klassen) [inline fly/commands/builds.go]: glad that you found this logic! but I think `(A && B) || A == A` (by set theory, or by doing the truth table), so you can just say `startTime == zeroTime` here!

**Comment by pivotal-jamie-klassen** [comment]
Also, the DCO bot will complain if your commit does not have a `Signed-off-by` line that matches the commit's author. This is part of our contributor agreement.

**Comment by iamjarvo** [comment]
Thanks for the feedback, I will address your feedback

**Comment by iamjarvo** [comment]
@pivotal-jamie-klassen made the changes you requested

**Comment by pivotal-jamie-klassen** [comment]
release note in #4565

**Pull request merged**